### ISSUE TITLE
Record audio of different devices

### ISSUE THREAD

**Issue by FlyingPersian** [tolga9009/elgato-gchd#50]
Hello,

I just installed the software on my Ubuntu laptop and recording in general works fine. I want to use it to record things I play on my windows desktop. I would like to hear everything via my USB headset and use it to talk to my mates via Discord. On Windows, I have setup Discord to use my headset as in- and output device. On Windows, I can either set the output audio device to Elgato, my speakers or headset. Obviously, I'd like to set that to my headset as well, but then I do not have any sound on the recording on my laptop. Is there a workaround for that so that I can hear ingame sounds and discord on my headset and record it on my laptop via Elgato Game Capture HD?  

Edit: I don't want to record on my PC directly (with nVidia Shadowplay, Fraps etc) because my CPU usage is already at 100% and sometimes updates break my game so that it will crash once in a while due to CPU usage. Thus, I want to keep CPU usage to a minimum.

**Comment by DusteDdk** [comment]
Can you rephrase your question, I am not sure what you want to do.

What is connected to what?
What runs what?
Where is the sound coming from?
Where should it go ?
